From e37b49559bd82cdbde4d28cc02da9a8623a39880 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 23 Apr 2018 19:21:27 +0200
Subject: [PATCH] doc: clarify the intended audience of CGROUP_DELEGATION.md

There was some confusion about who is the intended audience. Let's
clarify that early on, to ensure people aren't disappointed.

(cherry picked from commit 1e46eb5971f83d57b320082862cc1174120e1d9c)
---
 doc/CGROUP_DELEGATION.md | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/CGROUP_DELEGATION.md b/doc/CGROUP_DELEGATION.md
index 8eded2ebb3..619c871212 100644
--- a/doc/CGROUP_DELEGATION.md
+++ b/doc/CGROUP_DELEGATION.md
@@ -1,5 +1,8 @@
 # Control Group APIs and Delegation
 
+*Intended audience: hackers working on userspace subsystems that require direct
+cgroup access, such as container managers and similar.*
+
 So you are wondering about resource management with systemd, you know Linux
 control groups (cgroups) a bit and are trying to integrate your software with
 what systemd has to offer there. Here's a bit of documentation about the
-- 
2.17.0

